packages/terra-clinical-item-collection/tests/nightwatch/item-collection/clinical-item-collection-spec.js

```diff
@@ -3,71 +3,50 @@
 
     const width = screenWidth(browser);
     if (width <= browser.globals.breakpoints.small[0]) {
-      browser.expect.element('#ItemCollection').to.have.attribute('data-terra-clinical-item-collection-list-view');
+      browser.expect.element('#ItemCollection ul').to.have.attribute('data-terra-clinical-item-collection-list-view');
     } else {
-      browser.expect.element('#ItemCollection').to.have.attribute('data-terra-clinical-item-collection-table-view');
+      browser.expect.element('#ItemCollection table').to.have.attribute('data-terra-clinical-item-collection-table-view');
     }
   },
 
-  'Displays the correct collection responsive to window size for medium breakpoint': (browser) => {
-    browser.url(`${browser.launchUrl}/#/tests/item-collection-tests/medium-breakpoint`);
-    browser.execute("document.getElementById('siteApplication').style.padding = 0;");
-
-    const width = screenWidth(browser);
-    if (width <= browser.globals.breakpoints.medium[0]) {
-      browser.expect.element('#ItemCollection').to.have.attribute('data-terra-clinical-item-collection-list-view');
-    } else {
-      browser.expect.element('#ItemCollection').to.have.attribute('data-terra-clinical-item-collection-table-view');
-    }
+  'Displays a selectable item collection': (browser) => {
+    browser.url(`${browser.launchUrl}/#/tests/item-collection-tests/selectable`)
+      .expect.element('#ItemCollection').to.be.present;
   },
 
-  'Displays the correct collection responsive to window size for large breakpoint': (browser) => {
-    browser.url(`${browser.launchUrl}/#/tests/item-collection-tests/large-breakpoint`);
-    browser.execute("document.getElementById('siteApplication').style.padding = 0;");
-
-    const width = screenWidth(browser);
-    if (width <= browser.globals.breakpoints.large[0]) {
-      browser.expect.element('#ItemCollection').to.have.attribute('data-terra-clinical-item-collection-list-view');
-    } else {
-      browser.expect.element('#ItemCollection').to.have.attribute('data-terra-clinical-item-collection-table-view');
-    }
+  'Returns child key for click when onSelect is given to a selectable item collection': (browser) => {
+    browser.url(`${browser.launchUrl}/#/tests/item-collection-tests/selectable`);
+    browser.expect.element('#item1').to.be.present;
+    browser.click('#item1');
+    browser.expect.element('#selected-key').text.to.contain('1');
   },
 
-  'Displays the correct collection responsive to window size for huge breakpoint': (browser) => {
-    browser.url(`${browser.launchUrl}/#/tests/item-collection-tests/huge-breakpoint`);
-    browser.execute("document.getElementById('siteApplication').style.padding = 0;");
+  'Returns child key for ENTER when onSelect is given to a selectable item collection': (browser) => {
+    browser.url(`${browser.launchUrl}/#/tests/item-collection-tests/selectable`);
+    browser.expect.element('#item3').to.be.present;
```

Comment:
if added a `.to.equal(true)` here can remove the ignore of the linter rule of no-unused-expressions

Reply:
To be honest, I think the format that it's in here is more readable, and that adding that boolean check could make the code a little more confusing. I would rather just disable the linter here, as we have a good reason for doing it.